papeleta-pdf-extractor/src/config/2020/base-config.ts

```diff
@@ -0,0 +1,223 @@
+export interface BaseRow {
```

Comment:
Would there be more cases where we want to extend BaseRow?

Reply:
Not that I can see as of yet. If you think we should continue with this effort then it's a matter of taking a look at the municipal configs to see if this `BaseRow` would be extended later.